dokka-integration-tests/utilities/src/main/kotlin/org/jetbrains/dokka/it/gradleRunnerUtils.kt

```diff
@@ -1,4 +1,7 @@
-package org.jetbrains.dokka.dokkatoo.utils
+/*
+ * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+package org.jetbrains.dokka.it
```

Comment:
why it's placed in utilities and not in `gradle` subproject?

Reply:
Good point, I'll move it